main.cpp

```diff
@@ -209,6 +236,8 @@ struct School
     void addStudentToClass(Teacher teacher, std::string studentName);
     // add course to full course catalog
     void addCourse(std::string courseName, int gradeLevel = 9);
+    // new member function
+    School returnSchool(int threshold, int startingValue);
```

Comment:
a School instance will return another School?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a School instance...
- [ ] revise